kubevirt-cdi/step2.md

```diff
@@ -0,0 +1,68 @@
+#### Introduction on Containerized Data Importer
+
+[CDI](https://github.com/kubevirt/containerized-data-importer) is an utility designed to import Virtual Machine images for use with Kubevirt.
+
+At a high level, a PersistentVolumeClaim (PVC) is created. A custom controller watches for importer specific claims, and when discovered, starts an import process to create a raw image named *disk.img* with the desired content into the associated PVC.
+
+#### Install the CDI
+
+We will first explore each component and install them. In this exercise we create a hostpath provisioner and storage class. Also we will deploy the CDI component using the Operator.
+
+`wget https://raw.githubusercontent.com/kubevirt/kubevirt.github.io/master/labs/manifests/storage-setup.yml
+kubectl create -f storage-setup.yml
+export VERSION=$(curl -s https://github.com/kubevirt/containerized-data-importer/releases/latest | grep -o "v[0-9]\.[0-9]*\.[0-9]*")
+kubectl create -f https://github.com/kubevirt/containerized-data-importer/releases/download/$VERSION/cdi-operator.yaml
+kubectl create -f https://github.com/kubevirt/containerized-data-importer/releases/download/$VERSION/cdi-cr.yaml`{{execute}}
+
+Review the "cdi" pods that were added.
+
+`kubectl get pods -n cdi`{{execute}}
```

Comment:
immediately after running that command, it returns some containers are in error mode, as it is normal, a warning to the user should be placed for him/her not to worry about it.